data-otservbr-global/scripts/talkactions/gm/create_item.lua

```diff
@@ -5,12 +5,15 @@ local invalidIds = {
 local createItem = TalkAction("/i")
 
 function createItem.onSay(player, words, param)
-	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GOD then
+	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GAMEMASTER then
```

Comment:
GMs shouldn't be able to create items
```suggestion
	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GOD then
```